src/ecrc-frontend/src/components/base/termsOfUse/TermsOfUse.js

```diff
@@ -5,14 +5,34 @@ import { Button } from "../button/Button";
 
 import "./TermsOfUse.css";
 
-export default function TermsOfUse({ onClick }) {
+export default function TermsOfUse({
+  onContinueClick,
+  checkFirstBox,
+  checkSecondBox,
+  termOfUseOnScroll,
+  continueBtnEnabled
+}) {
   const button = {
     label: "Continue",
-    buttonStyle: "btn btn-primary",
-    buttonSize: "btn btn-sm",
+    buttonStyle: "btn ecrc_go_btn",
+    buttonSize: "btn ",
+    type: "submit",
+    disabled: !continueBtnEnabled
+  };
+
+  const cancelButton = {
+    label: "Cancel and Exit",
+    buttonStyle: "btn ecrc_accessary_btn",
+    buttonSize: "btn",
     type: "submit"
   };
 
+  const onCancelClicked = () => {
+    window.location.replace(
```

Comment:
are we specifically using `window.location.replace` here so that the current page is not stored in session history? is it a requirement to prevent user from being able to navigate back using back button?

Reply:
just wondering why we arent using a `<Redirect />` here

Reply:
as I do not know redirect. I remember Alan did some routing staff. I will update according to our app current implementation.

Reply:
Let's leave it for now. We need to confirm with Adrianna.